### PR TITLE
Cleanup and refactor

### PR DESCRIPTION
**Pull request by seavor**

### CODE REVIEW

**Comment by ZeldaZach** [comment]
cc: @jchamish @mixmix @Shmiddty @ebbit1q

**Comment by ZeldaZach** [comment]
cc: @jkachurek

File: webclient/src/components/ThreePaneLayout/ThreePaneLayout.css [hunk 1]
```diff
@@ -22,12 +22,19 @@
   flex-shrink: 0;
 }
 
-.three-pane-layout .grid-main__top.fixedHeight {
-  height: 50%;
+.three-pane-layout .grid-main__bottom {
```
Comment (mixmix) [inline webclient/src/components/ThreePaneLayout/ThreePaneLayout.css:25]: This selector is already defined below. Merge with that

**Review by mixmix — APPROVED** [review]
Seems like a fairly straightforward renaming.

Not sure why there are style changes in here as well. For future reference that makes this harder to review - I've not spun this up to see how it looks

**Comment by seavor** [comment]
> Seems like a fairly straightforward renaming.
> 
> Not sure why there are style changes in here as well. For future reference that makes this harder to review - I've not spun this up to see how it looks

Fair point, this is the result of a live coding session we did, so it has a few things going on.

**Review by ZeldaZach — APPROVED** [review]

**Pull request merged**